CommunityFridgeMapApi/dependencies/python/notification.py

```diff
@@ -0,0 +1,71 @@
+from db import DB_Response
+import re
+import boto3
+from botocore.exceptions import ClientError
+
+#https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses/client/send_email.html
+
+# def sendNotification(message, userId, lastNotified, email, phoneNumber):
+
+
+def send_sms(phone_number:str=None, message:str=None, region:str='us-east-1'):
+    
+    sns = boto3.client(
+        "sns",
+        region_name=region
+    )
+    if(not is_valid_phone_number(phone_number)):
+        return
+
+    response = sns.publish(
+        PhoneNumber=phone_number,
+        Message = message
+    )
+
+    return
```

Comment:
can you check what the response of sns.publish is and return if sns.publish was successful or not

Reply:
sns.publish returns a unique message ID identifier, and a sequence number index.  The sequence number seems less relevant for our purposes.

This update catches any errors and returns the status of message send, error encountered, message ID, and sequence number